host/3.0/buster/amd64/java/java8/mvn-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/sh -eu
```

Comment:
Looks the same file. We might consider to make it one place. Let's DRY it. e.g. before the build, we might copy the  file to the target directory.

Reply:
dried as suggested, please check.